java-support/docs/src/modules/java/pages/crdt.adoc

```diff
@@ -158,7 +158,7 @@ A map of PNCounter values. This exposes the current value of the PNCounters dire
 
 == Registering the entity
 
-Once you've created your entity, you can register it with the link:{attachmentsdir}/api/io/cloudstate/javasupport/CloudState.html[`CloudState`] server, by invoking the link:++api/io/cloudstate/javasupport/CloudState.html#registerEventSourcedEntity-java.lang.Class-com.google.protobuf.Descriptors.ServiceDescriptor-com.google.protobuf.Descriptors.FileDescriptor...-++[`registerCrdtEntity`] method.
+Once you've created your entity, you can register it with the link:{attachmentsdir}/api/io/cloudstate/javasupport/CloudState.html[`CloudState`] server, by invoking the link:{attachmentsdir}/api/io/cloudstate/javasupport/CloudState.html[`CloudState`] server, by invoking the `registerCrdtEntity` method.
```

Comment:
Good catch. Must have missed this one with the special `++` quoting. I think we can still have the link to the specific method (which is incorrectly `registerEventSourcedEntity` anyway). I also notice that these method links will be broken now because we're generating Java 11 javadoc, which uses parentheses instead. And sadly, the antora link checking doesn't cover these links — something we should address.

My suggestion for this link is to use:

```suggestion
Once you've created your entity, you can register it with the link:{attachmentsdir}/api/io/cloudstate/javasupport/CloudState.html[`CloudState`] server, by invoking the link:{attachmentsdir}++/api/io/cloudstate/javasupport/CloudState.html#registerCrdtEntity(java.lang.Class,com.google.protobuf.Descriptors.ServiceDescriptor,com.google.protobuf.Descriptors.FileDescriptor...)++[`registerCrdtEntity`] method.
```